packages/appsync-modelgen-plugin/src/__tests__/visitors/appsync-swift-visitor.test.ts

```diff
@@ -41,13 +41,27 @@ describe('AppSyncSwiftVisitor', () => {
         public let id: String
         public var name: String?
         public var bar: String?
+        public var createdAt: Temporal.DateTime?
+        public var updatedAt: Temporal.DateTime?
         
-        public init(id: String = UUID().uuidString,
-            name: String? = nil,
+        public init(name: String? = nil,
             bar: String? = nil) {
+          self.init(,
```

Comment:
here and throughout the generated public initializers, there's an extraneous comma :)